test/action_controller/json_api/transform_test.rb

```diff
@@ -77,7 +77,7 @@ def test_render_resource_with_transform
           expected = {
             'Data' => {
               'Id' => '1337',
-              'Type' => 'Posts',
+              'Type' => 'posts',
```

Comment:
Yeah. But, I'm not sure if this pr can be non breaking.  To preserve the test, I could add config that would make it behave the same as before this PR